deployment/deployment.json

```diff
@@ -1186,7 +1186,7 @@
         "status": "dev",
         "versions": {
           "schema": "2.0.1",
-          "subgraph": "1.1.4",
+          "subgraph": "1.2.0",
```

Comment:
A patch should only update the last number: https://semver.org/#summary

Reply:
Maybe I should reword the title of the PR instead. I feel the changes are substantial enough to warrant a minor version increase.

Reply:
I misread the title, I thought it said #patch . I guess the way I think of it is that since nothing was added to the schema there were no functionality changes. But honestly I am not totally sure